modules/@ergonode/authentication/src/components/Forms/PasswordRecoveryCheckEmail.vue

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <LoginForm
+        :submit-title="$t('authentication.forms.checkEmail.redirectTo')"
+        @submit="onRedirectToLogin">
+        <template #header>
+            <Fab @click.native="onBack">
+                <template #icon="{ color }">
+                    <IconArrowPointer :fill-color="color" />
+                </template>
+            </Fab>
+            <FormHeader :text="$t('authentication.forms.newPassword.title')" />
+        </template>
+        <template #body>
+            <FormParagraph :text="$t('authentication.forms.checkEmail.info')" />
+            <p class="recovery-form__info">
+                <span
+                    v-text="$t('authentication.forms.checkEmail.infoText')" />
+                <span
+                    v-text="$t('authentication.forms.checkEmail.infoLink')"
```

Comment:
I've mentioned earlier about using `LinkButton`

Reply:
`LinkButton` is incorrect, because it has a different style and layout. 
Alternatively, we can make a component that inserts links in the middle of the paragraph in the same format without breaking the lines. This is not part of this task. I will prepare a separate task if needed.

Reply:
If the part of code is used in this task and it requires new component which is not present in design system, we should create it in this task, cuz it's part of it, or before making this task everything should be prepared.